coda/coda_mdstore/views.py

```diff
@@ -589,6 +589,13 @@ def bagURLList(request, identifier, html=False):
         return render(request, 'mdstore/bag_files_download.html',
                       {'links': sorted(transList)})
 
+    if download:
+        zip_filename = 'bag-' + identifier.replace('/', '_') + '.zip'
```

Comment:
I'm not particularly fond of this file naming convention, though I suppose it doesn't really matter. What do people (@vphill @madhulika95b @somexpert ) think about just having the coda meta_id like coda101.zip or dropping the 'bag-' prefix?

Reply:
I would prefer the bag name to be the `metaid + '.zip'` like `coda101.zip`